tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Builders/OrderSummaryDataBuilder.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Data;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.EntityBuilder;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Builders
+{
+    internal sealed class OrderSummaryDataBuilder
+    {
+        private List<OrderEntity> _OrderEntities;
+        private List<OrderItemEntity> _OrderItemEntities;
+        private List<ServiceRecipientEntity> _serviceRecipientEntites;
+
+        private OrderSummaryDataBuilder( )
+        {
+            _OrderEntities = new List<OrderEntity>();
+            _OrderItemEntities = new List<OrderItemEntity>();
+            _serviceRecipientEntites = new List<ServiceRecipientEntity>();
+        }
+
+        public static OrderSummaryDataBuilder CreateOrderSummaryData(string orderId = "C000016-01") => 
+            new OrderSummaryDataBuilder()
+                .AddOrderEntity(
+                    OrderEntityBuilder.Create()
+                        .WithOrderStatusId(2)
```

Comment:
Could make use of an enum here to provide context surrounding the magic number. It is no longer submitted / unsubmitted. But complete / incomplete